### DISCUSSION TITLE
Error at export to ONNX-FINN (Caching of output shapes is required to export QuantConvNd)

### DISCUSSION THREAD

**Discussion by balabengba** [Xilinx/finn#424] [Q&A]
Hi everyone, I'm trying to implement segmentation NN as described in the FINN-R paper with Brevitas and FINN. I describe the NN only by **QuantConv2d**.
When I export to ONNX-FINN, I get this error message:
**RuntimeError: Caching of output shapes is required to export QuantConvNd**
This is my export command:
`bo.export_finn_onnx(model , export_path=onnx_filename, input_shape=(1, 3, 512, 512) )`
Could you help me with that?
    torch : 1.10.0
    brevitas: 0.7.0
Thanks in advance for your help.

**Comment by julienmas** [comment]
Hi, did you find a solution?

**Reply by MFJI** [reply]
When I met this problem, my solution is remove the functions of "self.conv=qnn.QuantConv2d" that won't be used later on.

**Comment by MFJI** [comment]
When I met this problem, my solution is remove the functions of "self.conv=qnn.QuantConv2d" that won't be used later on.
